test/header/CMakeLists.txt

```diff
@@ -77,12 +76,8 @@ macro (seqan3_header_test component header_base_path)
     file (WRITE "${PROJECT_BINARY_DIR}/header_test_src/empty.cpp" "")
 
     add_executable (${target} ${PROJECT_BINARY_DIR}/header_test_src/empty.cpp)
-    target_compile_options (${target} PRIVATE ${cxx_flags_list} ${SEQAN3_TEST_CXX_FLAGS})
-    target_compile_definitions (${target} PRIVATE ${SEQAN3_DEFINITIONS} ${SEQAN3_TEST_DEFINITIONS})
-    target_include_directories (${target} PRIVATE ${SEQAN3_INCLUDE_DIRS} ${SEQAN3_TEST_INCLUDE_DIRS})
-    target_link_libraries (${target} ${SEQAN3_LIBRARIES} ${SEQAN3_TEST_LIBRARIES})
+    target_link_libraries (${target} seqan3_unit_test)
```

Comment:
where do we actually tell cmake to use the previously defined "library"?

Reply:
`seqan3_unit_test` inherits from `seqan3::seqan3`

Reply:
yep I saw that later on